src/Personal/actionCreator.js

```diff
@@ -0,0 +1,144 @@
+import { BACKEND_URL } from 'react-native-dotenv';
+import { restoreTransactions } from '../Stats/actionCreator';
+
+const moment = require('moment');
+
+const accessToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6Im1pa2UiLCJlbWFpbCI6Im1pa2VAZ21haWwuY29tIiwiaWQiOiI1ZTEwYmMyNDcwMWZiZDQ4OTczNzdkOWYiLCJpYXQiOjE1NzgzNjQxOTQsImV4cCI6MTU4MDk1NjE5NH0.KiAzpui0f9-uwrAGVghNwukkEPWgeHHGkZH4sEm_v6o';
+// const accessToken = '';
+
+export const actionType = {
+  BACKUP_START: 'BACKUP_START',
+  BACKUP_SUCCESSFUL: 'BACKUP_SUCCESSFUL',
+  BACKUP_FAILED: 'BACKUP_FAILED',
+  SIGNUP_START: 'SIGNUP_START',
+  SIGNUP_SUCCESSFUL: 'SIGNUP_SUCCESSFUL',
+  SIGNUP_FAILED: 'SIGNUP_FAILED',
+  RESTORE_START: 'RESTORE_START',
+  RESTORE_SUCCESSFUL: 'RESTORE_SUCCESSFUL',
+  RESTORE_FAILED: 'RESTORE_FAILED',
+};
+
+export function backupStart() {
+  return {
+    type: actionType.BACKUP_START,
+  };
+}
+
+export function backupSuccessful(timestamp) {
+  return {
+    type: actionType.BACKUP_SUCCESSFUL,
+    payload: timestamp,
+  };
+}
+
+export function backupFailed(errorMsg) {
+  return {
+    type: actionType.BACKUP_FAILED,
+    payload: errorMsg,
+  };
+}
+
+export function signupStart() {
+  return {
+    type: actionType.SIGNUP_START,
+  };
+}
+
+export function signupSuccessful(userInfo) {
+  return {
+    type: actionType.signupSuccessful,
```

Comment:
isn't it SIGNUP_SUCCESSFUL?
```suggestion
    type: actionType.SIGNUP_SUCCESFUL,
```